demo/hyproof-demo.md

```diff
@@ -141,5 +146,25 @@ The truth of this data could then be confirmed against the cryptographic data wi
 
 In this way `Heidi` and `Emma` keep each other honest, and the `Regulator` keeps them both honest.
 
+### 4. Revoking the certificate
```

Comment:
Sorry , I can't point this to the right line but this is about line 114 .. from

4. `Emma` submits this embodied CO2 value to the ledger to be added to the certificate that `Heidi` previously **intitiated**


to 

4. `Emma` submits this embodied CO2 value to the ledger to be added to the certificate that `Heidi` previously **initiated**